core/src/main/java/org/testcontainers/containers/output/FrameConsumerResultCallback.java

```diff
@@ -45,9 +58,13 @@ public void onNext(Frame frame) {
             if (outputFrame != null) {
                 Consumer<OutputFrame> consumer = consumers.get(outputFrame.getType());
                 if (consumer == null) {
-                    LOGGER.error("got frame with type " + frame.getStreamType() + ", for which no handler is configured");
+                    LOGGER.error("got frame with type {}, for which no handler is configured", frame.getStreamType());
                 } else {
-                    consumer.accept(outputFrame);
+                    if (frame.getStreamType() == StreamType.RAW) {
```

Comment:
both methods have `if (outputFrame != null) { if(utf8String != null && !utf8String.isEmpty()) { ... } }`, I would move it here